generic/tests/fillup_disk.py

```diff
@@ -23,13 +23,21 @@ def run(test, params, env):
     session = vm.wait_for_login(timeout=login_timeout)
     session2 = vm.wait_for_serial_login(timeout=login_timeout)
 
+    mount_lv = params.get("mount_lv", "no")
+    mount_cmd = params.get("mount_cmd")
     fillup_timeout = int(params.get("fillup_timeout"))
     fillup_size = int(params.get("fillup_size"))
     fill_dir = params.get("guest_testdir", "/tmp")
     filled = False
     number = 0
 
     try:
+        if mount_lv == "yes":
+            error.context("Mounting filesystem by command '%s'" % mount_cmd,
+                          logging.info)
+            s, o = session.cmd_status_output(mount_cmd)
+            if s != 0 and "already mounted" not in o:
```

Comment:
thanks for your commnets. for sub test "fillup_disk" and "ioquit", may be used in other test case that no need to mount lv, so add "mount_lv=yes/no" in test and not check the mount point